### ISSUE TITLE
Problem downloading mouse database

### ISSUE THREAD

**Issue by abiadak** [charite/jannovar#473]
I can't download any of mouse databases, I obtain the following error:
java -jar jannovar-cli-0.33.jar download -d mm10/ensembl
Options
JannovarDownloadOptions [downloadDir=data, getDataSourceFiles()=[bundle:///default_sources.ini], isReportProgress()=true, getHttpProxy()=null, getHttpsProxy()=null, getFtpProxy()=null, geneIdentifiers=[], outputFile=]
Downloading/parsing for data source "mm10/ensembl"
ERROR: Section mm10/ensembl does not contain key table_gene_main
de.charite.compbio.jannovar.datasource.InvalidDataSourceException: Section mm10/ensembl does not contain key table_gene_main
	at de.charite.compbio.jannovar.datasource.DataSource.checkURLs(DataSource.java:100)
	at de.charite.compbio.jannovar.datasource.EnsemblDataSource.<init>(EnsemblDataSource.java:24)
	at de.charite.compbio.jannovar.datasource.DataSourceFactory.getDataSource(DataSourceFactory.java:95)
	at de.charite.compbio.jannovar.cmd.download.DownloadCommand.run(DownloadCommand.java:43)
	at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:67)

I've no problem downloading human databases, as hg19/refseq for example.

**Comment by visze** [comment]
Thank you for your ticket. Yes you are right. We have a lot of problems right now building transcript databases for two reasons:
* Lots of URLs changed and we need to provide the correct ones for download. 
* We introduced with an improvement of human MT in version 0.33 annotation a horrible bug so that other species do not work properly.

I recently created a quick fix (see #472) for both issues. But this is a workaround and still mouse ensembl databases do not work (but the mouse refseq). In the future we will completely change the way we generate transcript data. We will make data releases of already serialized transcript databases so that the user do not have to generate them by their own. But we will provide the option to generate own databases (in a simpler way). But this needs a complete refactoring of the database module.

For now I would recommend to use the version 0.32
